parsec/backend/postgresql/factory.py

```diff
@@ -31,6 +33,15 @@ async def components_factory(config: BackendConfig, event_bus: EventBus):
         event_bus,
     )
 
+    async def _send_event(
+        event: BackendEvent, conn: triopg._triopg.TrioConnectionProxy = None, **kwargs
```

Comment:
`Optional[triopg._triopg.TrioConnectionProxy]` ?

and the `triopg._triopg.TrioConnectionProxy` seems like a poor path I think we should add an issue in triopg about this ;-)

Reply:
:+1: